apps/patient/src/components/statusV2/Step.tsx

```diff
@@ -9,36 +9,18 @@ export interface StepProps {
 }
 
 export const Step = ({ color, icon, complete, iconProps }: StepProps) => {
+  const uiColor = complete
```

Comment:
nit: nested ternaries like this are so hard for me to digest! maybe extract this into a `getUiColor` function and refactor it into something a lil bit easier to read?

Reply:
gj removing that duplicated logic in any case though